core/SConscript.firmware

```diff
@@ -557,6 +557,7 @@ if FROZEN:
     ))
 
     SOURCE_PY.extend(Glob(SOURCE_PY_DIR + 'trezor/wire/*.py'))
+    SOURCE_PY.extend(Glob(SOURCE_PY_DIR + 'trezor/wire/codec/*.py'))
```

Comment:
The codec specific files are separated so that they can be easily ommited for THP prod builds. When the THP implementation is added, it could look like this:
```
if THP:
   SOURCE_PY.extend(Glob(SOURCE_PY_DIR + 'trezor/wire/thp/*.py'))
else:
   SOURCE_PY.extend(Glob(SOURCE_PY_DIR + 'trezor/wire/codec/*.py'))
   ```

Reply:
Note: The condition `or PYOPT == '0'` is thee because the codec is used on DebugLink even with THP builds.